### PR TITLE
Adding Vuefity to the application

### PR DESCRIPTION
**Pull request by Prashanga**
## Rationale

Adding Vuetify, a material design component framework to the application
The documentation is available at: https://vuetifyjs.com/en/getting-started/quick-start/

## Advice for Reviewers & Testing Notes

-The app should still work fine. 
-This PR only adds the framework to the application. Its use cases will be added in another PR

## Screenshots:

na

## Checklist
- [x ] PR your own code before you assign reviewers

### CODE REVIEW

**Comment by promie** [comment]
Can we also test to see Iif the global scss variable still works?

**Comment by Prashanga** [comment]
I've changed the navbar color from global sass variables. Please check it

File: Web/client/src/assets/scss/_variables.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@ $label: #000000;
 $secondaryLable: #3c3c43;
 $lightGray: #aaaaaa;
 $darkGray: #555555;
+$navbar-background-color: pink;
```
Comment (promie) [inline Web/client/src/assets/scss/_variables.scss:8]: Should we make the name of this variable more generic? Surely we wont be using the pink colour only in the Navbar?

**Comment by Prashanga** [comment]
webpack.config.js  hasn't been changed in this commit. So it should work as before

File: Web/client/src/assets/scss/_variables.scss (hunk 1 above)
Comment (Prashanga) [reply Web/client/src/assets/scss/_variables.scss:8]: Surely, It's just a demo of sass working. My next PR will be for a new navbar. I'll update the colors then.

**Pull request merged**